src/Sylius/Bundle/ResourceBundle/DependencyInjection/Driver/DoctrineORMDriver.php

```diff
@@ -21,42 +23,41 @@
  * @author Arnaud Langlade <aRn0D.dev@gmail.com>
  * @author Gonzalo Vilaseca <gvilaseca@reiss.co.uk>
  */
-class DoctrineORMDriver extends AbstractDatabaseDriver
+class DoctrineORMDriver extends AbstractDriver
 {
     /**
      * {@inheritdoc}
      */
-    public function getSupportedDriver()
+    protected function getRepositoryDefinition(ResourceMetadataInterface $metadata, ContainerBuilder $container)
     {
-        return SyliusResourceBundle::DRIVER_DOCTRINE_ORM;
-    }
-
-    /**
-     * {@inheritdoc}
-     */
-    protected function getRepositoryDefinition(array $classes)
-    {
-        $reflection = new \ReflectionClass($classes['model']);
+        $reflection = new \ReflectionClass($metadata->getClass('model'));
         $translatableInterface = 'Sylius\Component\Translation\Model\TranslatableInterface';
         $translatable = (interface_exists($translatableInterface) && $reflection->implementsInterface($translatableInterface));
 
-        $repositoryKey = $this->getContainerKey('repository', '.class');
-        $repositoryClass = $translatable
-            ? 'Sylius\Bundle\TranslationBundle\Doctrine\ORM\TranslatableResourceRepository'
-            : new Parameter('sylius.orm.repository.class');
+        $repositoryClassParameter = $metadata->getServiceId('repository').'.class';
+
+        $repositoryClass = $translatable ?
+            new Parameter('sylius.doctrine.orm.translatable_repository.class') :
+            new Parameter('sylius.doctrine.orm.repository.class')
+        ;
 
-        if ($this->container->hasParameter($repositoryKey)) {
-            $repositoryClass = $this->container->getParameter($repositoryKey);
+        if ($container->hasParameter($repositoryClassParameter)) {
```

Comment:
If container dont have this parameter then EntityManager->getRepository will return dafault repository class but not  `%ylius.doctrine.orm.translatable_repository.class%` or `%sylius.doctrine.orm.repository.class%`
Sylius\Bundle\ResourceBundle\EventListener\LoadODMMetadataSubscriber
```php 
if (isset($class['repository'])) {
                        $metadata->setCustomRepositoryClass($class['repository']);
                    }
```